python/ml_wrappers/model/image_model_wrapper.py

```diff
@@ -22,14 +26,13 @@
 try:
     import torch.nn as nn
 except ImportError:
-    module_logger.debug('Could not import torch, required if using a PyTorch model')
+    module_logger.debug("Could not import torch, required if using a PyTorch model")
```

Comment:
Was this auto-changed somehow?  I think we try to stick to single quotes everywhere across all RAI repos for strings in python code.